packages/components/src/components/select-native/select-native.tsx

```diff
@@ -29,8 +29,8 @@ type TSelectNativeOptions = {
 };
 
 type TListItem = {
-    text: string;
-    value: string;
+    text?: string;
```

Comment:
Are both text and value optional in a list item? Can u recheck this typing please?

Reply:
I had to change it, because there were errors while passing list_items. 
I think, we should better refactor types, because we use select native with dropdown, but the props types are different